### DISCUSSION TITLE
Pre Windows 10 ANSI console support

### DISCUSSION THREAD

**Discussion by mbostleman** [spectreconsole/spectre.console#808] [Q&A] [answered]
I have a net5 console app that is using Spectre 0.44.0 and it's throwing an exception on Windows Server 2012 which is derived from Windows 8 which is obviously prior to Windows 10 at which point ANSI started being supported by default. There was this comment on a Hacker News post a year or so ago:

"If the terminal don't support ANSI escape sequences, Spectre.Console will fall back to using the System.Console API with the standard 4-bit color palette. Everything will continue working, but less pretty of course."

I would love it if I could make that happen instead of getting the NotSupportedException I'm seeing now. Is that possible?

The exception is being thrown at Spectre.Console.ListPrompt.

**Comment by nils-a** [comment, marked as answer]
@mbostleman I guess you're running against https://github.com/spectreconsole/spectre.console/issues/495

**Reply by mbostleman** [reply]
Yep, that's exactly it - Server 2012, ListPrompt, NotSupportedException. Ok, guess it's time to pivot. Thanks for the pointer!
